tasks/main.yml

```
@@ -73,12 +73,19 @@
   when: tlog_use_sssd
   notify: Handler tlog_handler restart sssd
 
-- name: Check with-files-domain feature exists
+- name: Check appropriate authselect features exists
   command: authselect list-features sssd
   register: __tlog_authselect_features
   changed_when: false
 
-- name: Check if files domain is currently enabled
+- name: Enable authselect profile with tlog feature
+  command: authselect select sssd with-tlog --force
```

```suggestion
  command: authselect select sssd with-tlog --force
  changed_when: true
```
This will definitely change the state of the system when the `when` conditions are met.